src/Module/taskapp.js

```diff
@@ -0,0 +1,128 @@
+const taskapp = () => {
+  class Task {
+    constructor() {
+      this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+      this.taskTitle = document.querySelector('#task-description');
+      this.taskStatus = document.querySelector('#author');
+      this.taskDisplay = document.querySelector('#display');
+      this.buttonAdd = document.querySelector('#add');
+      this.clearall = document.querySelector('#clearall');
+      this.checkboxStatus = document.querySelector('#checkboxid');
+      this.render();
+    }
+
+    addTask(id, title, status) {
+      this.tasks.push({ id, title, status });
+      localStorage.setItem('tasks', JSON.stringify(this.tasks));
+      this.render();
+    }
+
+    removeTask(index) {
+      this.tasks.splice(index, 1);
+      localStorage.setItem('tasks', JSON.stringify(this.tasks));
+      this.render();
+    }
+
+    editTask(target) {
```

Comment:
Good job with adding the editing task. But let's implement the edit behaviour as seen in the video as requested [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#how-to-build-the-to-do-list-website). 
>Your task is to build a website that displays a list that looks and behaves like the part of minimalist project capture in the video below.

Reply:
- I think your function for editing input has not been correctly implemented. just as @huseyinbiyik suggested, can you fix this?? 🌸 
When I try to edit, and click enter, the terms "edit" and "delete"  automatically add to the newly edited input. Here is what I mean 👇🏽 

<img width="657" alt="Screenshot 2023-02-03 at 23 02 29" src="https://user-images.githubusercontent.com/99246359/216749860-f18ac31d-00c4-489c-b35a-5ef1a3903d68.png">